lib/daru/io/exporters/excel.rb

```diff
@@ -23,32 +34,42 @@ class Excel < Base
         #
         #   Daru::IO::Exporters::Excel.new(df, "dataframe_df.xls").call
         #
-        # @todo The +opts+ parameter isn't used while creating the Excel Spreadsheet
-        #   yet. Implementing this feature will greatly allow the user to generate a
-        #   Spreadsheet of their choice.
-        def initialize(dataframe, path, **options)
+        # @example Writing to an Excel file with options
+        #   df = Daru::DataFrame.new([[1,2],[3,4]], order: [:a, :b])
+        #
+        #   #=> #<Daru::DataFrame(2x2)>
+        #   #      a   b
+        #   #  0   1   3
+        #   #  1   2   4
+        #
+        #   normal  = { color: :blue }
+        #   heading = { color: :red, weight: :bold }
+        #
+        #   Daru::IO::Exporters::Excel.new(df,
+        #     "dataframe_df.xls",
+        #     data_options: normal,
+        #     header_options: heading
+        #   ).call
```

Comment:
Currently, all arguments are given only in the `initialize()` method and none to `call()`.

```ruby
Daru::IO::Exporters::Excel.new(
  df,
  'path/file.xls',
  formatting: {header: {...}, data: {...}, index: {...}},
  display: {header: true, index: false, data: true}
)
```

Would it be better to rename `:header` to `:order`?